Marlin/src/HAL/STM32/tft/tft_spi.cpp

```diff
@@ -34,35 +34,25 @@ DMA_HandleTypeDef TFT_SPI::DMAtx;
 void TFT_SPI::Init() {
   SPI_TypeDef *spiInstance;
 
-  #if PIN_EXISTS(TFT_RESET)
-    OUT_WRITE(TFT_RESET_PIN, HIGH);
-    HAL_Delay(100);
-  #endif
-
-  #if PIN_EXISTS(TFT_BACKLIGHT)
-    OUT_WRITE(TFT_BACKLIGHT_PIN, HIGH);
-  #endif
```

Comment:
Why are these no longer needed?

Reply:
Because it's handled by TFT_IO class now. The TFT code for STM32 was outdated compared with LPC and STM32F1.